code/game/turfs/closed/wall/misc_walls.dm

```diff
@@ -110,3 +110,19 @@
 	sheet_type = /obj/item/stack/sheet/bronze
 	sheet_amount = 2
 	girder_type = /obj/structure/girder/bronze
+
+/turf/closed/wall/rock
+	name = "reinforced rock"
+	desc = "It has metal struts that need to be welded away before it can be mined."
```

Comment:
Feel like this should use the whole deconstruction description-tips we have for reinforced walls
ie
desc = "A rock wall reinforced by a metal support structure."
"The struts are <b>welded</b> together at the seams"

Reply:
Won't it already have the deconstruction tooltips by being a wall, or nah?

Reply:
its inherited from parent